tests/devsim_layer_test.sh

```diff
@@ -23,26 +23,31 @@ export LD_LIBRARY_PATH=${PWD}/../loader:${LD_LIBRARY_PATH}
 export VK_LAYER_PATH=${PWD}/../layersvt
 export VK_INSTANCE_LAYERS="VK_LAYER_LUNARG_device_simulation"
 
-# Test input datafile, and filename of output.
-FILENAME_01_IN="devsim_test1.json"
-FILENAME_01_GOLD="devsim_test1_gold.json"
-FILENAME_01_OUT="device_simulation_layer_test_1.json"
-
-export VK_DEVSIM_FILENAME="${PWD}/${FILENAME_01_IN}"
-
 #export VK_DEVSIM_DEBUG_ENABLE="1"
 #export VK_DEVSIM_EXIT_ON_ERROR="1"
 #export VK_LOADER_DEBUG="all"
 
 VKJSON_INFO="${PWD}/../libs/vkjson/vkjson_info"
 
-${VKJSON_INFO} > /dev/null
+#############################################################################
+# Test #1 input datafile, and filename of output.
+
+FILENAME_01_IN="devsim_test1.json"
+FILENAME_01_GOLD="devsim_test1_gold.json"
+FILENAME_01_RESULT="device_simulation_layer_test_1.json"
+FILENAME_01_STDOUT="device_simulation_layer_test_1.txt"
+
+export VK_DEVSIM_FILENAME="${PWD}/${FILENAME_01_IN}"
+${VKJSON_INFO} > ${FILENAME_01_STDOUT}
```

Comment:
Curious, why include ${PWD} above, but not here?

Reply:
Fair point, I think that may be historical.  There may have been more cd-ing going on previously, that would have made locating vkjson_info trickier.

Reply:
Make it consistent then?

Reply:
done